woocommerce/woocommerce-android
# Beta fix: pass entered product name into Product Name with AI

This PR includes the missing functionality as part of the beta testing reports made by @selanthiraiyan

Specifically, if the user enters a product name and then taps "Suggest a name", then the entered product name will be shown on the Product Name with AI bottom sheet's textbox.

### To test:
1. Start a new product creation with AI,
2. on the "Add your product name" screen, enter some text,
3. tap "Suggest a name",
4. Make sure the bottom sheet now shows the entered text on step 2.
